contracts/token/ERC721/Tatum721.sol

```diff
@@ -53,6 +54,23 @@ contract Tatum721 is ERC721Enumerable, ERC721URIStorage, AccessControlEnumerable
         return true;
     }
 
+    function mintWithCashback(address to, uint256 tokenId, string memory tokenURI, address[] authorAddresses, uint256[] cashbackValues) public returns (bool) {
+        require(hasRole(MINTER_ROLE, _msgSender()), "ERC721PresetMinterPauserAutoId: must have minter role to mint");
+        _mint(to, tokenId);
+        for (uint i=0;i<authorAddresses.length;i++){
+            _cashbacks[tokenId] = to;
+            _cashbacksValue[tokenId]=cashbackValues[i];
+            safeTransfer(authorAddresses[i],tokenId);    
```

Comment:
there should be no transfer here. this is mint method - tokens are minted to _to_ recipient, not to the author. authorAddresses represents the address, where cashback will be paid in the future transfers